dubbo-cluster/src/test/java/org/apache/dubbo/rpc/cluster/loadbalance/ConsistentHashLoadBalanceTest.java

```diff
@@ -19,22 +19,34 @@
 import org.apache.dubbo.rpc.Invoker;
 
 import org.junit.Assert;
-import org.junit.Ignore;
 import org.junit.Test;
 
+import java.util.HashMap;
 import java.util.Map;
 import java.util.concurrent.atomic.AtomicLong;
 
+@SuppressWarnings("rawtypes")
 public class ConsistentHashLoadBalanceTest extends LoadBalanceBaseTest {
-    @Ignore
+    
     @Test
     public void testConsistentHashLoadBalance() {
-        int runs = 10000;
-        Map<Invoker, AtomicLong> counter = getInvokeCounter(runs, ConsistentHashLoadBalance.NAME);
-        for (Map.Entry<Invoker, AtomicLong> entry : counter.entrySet()) {
-            Long count = entry.getValue().get();
-            Assert.assertTrue("abs diff should < avg", Math.abs(count - runs / (0f + invokers.size())) < runs / (0f + invokers.size()));
-        }
+    	int runs = 10000;
+		long invokerCount0 = 0;
```

Comment:
Just an thought on variable naming, is 'untouchedNodesCount' instead of invokerCount0  and touchedNodesHitMap looks more appropriate from intended functionality point of view ?

Reply:
That makes sense. I'm going to make some changes and give it a more reasonable name